pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -338,12 +338,6 @@ func GetContainerLogs(client DockerInterface, pod *v1.Pod, containerID kubeconta
 }
 
 var (
-	// ErrNoContainersInPod is returned when there are no containers for a given pod
-	ErrNoContainersInPod = errors.New("NoContainersInPod")
-
-	// ErrNoPodInfraContainerInPod is returned when there is no pod infra container for a given pod
-	ErrNoPodInfraContainerInPod = errors.New("NoPodInfraContainerInPod")
```

Comment:
Don't think we are going to use them, especially now that we are transitioning to CRI. Most of the code in this package (especially those mentioning "infra container") will be deprecated eventually.